preprocessors/autour.schema.json

```diff
@@ -32,20 +32,12 @@
             "pattern": "^https?://.*"
         },
         "places": {
-            "type": "array",
+            "type": "object",
             "title": "Places",
             "description": "The places on the map",
-            "items": {
+            "additionalProperties": {
```

Comment:
This isn't how additionalProperties works...it is either true or false to indicate if you can have additional properties in an object along with those explicitly listed in properties.